Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -243,6 +249,23 @@ protected Subject createEvidenceSubject(final CallbackProperties properties, fin
         return OpenSAML2ComponentBuilder.getInstance().createSubject(x509Name, publicKey);
     }
 
+    protected Conditions createConditions(final CallbackProperties properties) {
+        DateTime issueInstant = properties.getIssueInstant();
+        DateTime beginValidTime = properties.getSamlConditionsNotBefore();
+        DateTime endValidTime = properties.getSamlConditionsNotAfter();
+
+        if (issueInstant == null) {
+            issueInstant = new DateTime();
+        }
+
+        beginValidTime = setBeginValidTime(beginValidTime, issueInstant);
```

Comment:
This is where the  if statement should be for checking the property for allowing for corrected times.